pages/multi/[address]/index.tsx

```diff
@@ -49,9 +49,10 @@ const Multipage = () => {
         assert(state.chain.denom, "denom missing");
         const tempHoldings = await client.getAllBalances(address);
         setHoldings(tempHoldings);
-        const result = await getMultisigAccount(address, client);
-        setPubkey(result[0]);
-        setAccountOnChain(result[1]);
+        const [newPubkey, newAccountOnChain] = await getMultisigAccount(address, client);
+        assert(isMultisigThresholdPubkey(newPubkey), "pubkey is not multisig");
```

Comment:
The check is good, but some user visible error would be good. This case can easily happen if you enter the wrong address on the main screen.

Reply:
And I noticed that getMultisigAccount already does that check... So I should just make that check before this page is loaded, on the use multisig form to avoid getting there, and notify the user about that

Reply:
Right, so when the user opens the page via a wrong URL, the assertion is here fine I guess. But on the main page input there could be a nice check avoiding accidental usage of the wrong address.